### DISCUSSION TITLE
스탬프 적립 로직 개선

### DISCUSSION THREAD

**Discussion by youngh0** [woowacourse-teams/2023-stamp-crush#598] [General]
- 이벤트 기반 아키텍처 세미나 영상
[이벤트 기반 아키텍처](https://www.youtube.com/watch?v=b65zIH7sDug&list=PLgXGHBqgT2TsFnKRe3_kvFXDFUWxaUvQ2&index=32&t=519s)

**Comment by gitchannn** [comment]
애플리케이션 이벤트와 내부 이벤트가 잘 구분이 안가는데, 차이를 알려주세요

**Comment by gitchannn** [comment]
관련 참고영상

https://www.youtube.com/watch?v=TJUIkLFpgGo
https://www.youtube.com/watch?v=mXk0MNjlO7A

**Comment by gitchannn** [comment]
### 더 이야기한다고 결과가 나올 것 같지 않아서 잠정 보류 

### Stamp, Coupon은 어쩌면 EventListener로 분리할 수 있도록 하는 것이 아닐 수도 있다. 

### Stamp, VisitHistory는 분리하는 것이 괜찮을 것 같다.

### 각자 더 생각해오기로 함.
